lib/routes/sohu/mobile.ts

```diff
@@ -0,0 +1,209 @@
+    if (standardMatch) {
+        // 替换斜杠为横杠确保兼容性
+        const normalizedDateStr = standardMatch[1].replaceAll('/', '-');
+        const date = new Date(normalizedDateStr);
+        if (!Number.isNaN(date.getTime())) {
+            return date.toISOString();
+        }
+    }
+    //  "昨天23:59" 格式
+    if (cleanedStr.includes('昨天')) {
+        const timeMatch = cleanedStr.match(/昨天(\d{1,2}:\d{2})/);
+        if (timeMatch) {
+            const date = new Date();
+            date.setDate(date.getDate() - 1);
+            const [hours, minutes] = timeMatch[1].split(':');
+            date.setHours(Number.parseInt(hours), Number.parseInt(minutes), 0, 0);
+            return date.toISOString();
+        }
+    }
+    //  "n小时前" 格式
+    const hoursAgoMatch = cleanedStr.match(/(\d+)\s*小时前/);
+    if (hoursAgoMatch) {
+        const date = new Date();
+        date.setHours(date.getHours() - Number.parseInt(hoursAgoMatch[1]));
+        return date.toISOString();
+    }
+    //  "n分钟前" 格式
+    const minsAgoMatch = cleanedStr.match(/(\d+)\s*分钟前/);
+    if (minsAgoMatch) {
+        const date = new Date();
+        date.setMinutes(date.getMinutes() - Number.parseInt(minsAgoMatch[1]));
+        return date.toISOString();
+    }
+    // "04-29 23:32"
+    const shortDateMatch = cleanedStr.match(/^(\d{2})-(\d{2}) (\d{2}:\d{2})$/);
+    if (shortDateMatch) {
+        const [, month, day, time] = shortDateMatch;
+        const currentYear = new Date().getFullYear();
+        const date = new Date(`${currentYear}-${month}-${day} ${time}`);
+        if (!Number.isNaN(date.getTime())) {
+            return date.toISOString();
+        }
+    }
+    // "今天 HH:MM" 格式
+    if (cleanedStr.startsWith('今天')) {
+        const timeMatch = cleanedStr.match(/今天(\d{1,2}:\d{2})/);
+        if (timeMatch) {
+            const [hours, minutes] = timeMatch[1].split(':');
+            const date = new Date();
+            date.setHours(Number.parseInt(hours), Number.parseInt(minutes), 0, 0);
+            return date.toISOString();
+        }
+    }
+    //  "2025年04月29日 23:32"
+    const chineseDateMatch = cleanedStr.match(/^(\d{4})年(\d{2})月(\d{2})日 (\d{2}:\d{2})$/);
+    if (chineseDateMatch) {
+        const [, year, month, day, time] = chineseDateMatch;
+        const date = new Date(`${year}-${month}-${day} ${time}`);
+        if (!Number.isNaN(date.getTime())) {
+            return date.toISOString();
+        }
+    }
+    return null;
+}
+
+function getDescription($: cheerio.CheerioAPI) : string | null {
+    const selectors = [
+        '#articleContent',
+        'div.tpl-top-text-content-description',
+        'div.title-desc > div.desc > div',
+        '#videoTitle',
+    ];
+    for (const selector of selectors) {
+        const content = $(selector).first().html()?.trim();
+        if (content && content.length > 6) {
+            return content;
+        }
+    }
+    return null;
```

Comment:
In a short post (xtopic), visit the original URL in `.div.tpl-top-text-content-description` which you can reuse `getDescription` to obtain the original content.
In a video post, the video direct URL is available in the inner `div` of `#videoPlayer` which you can provide to the user.
In a gallery post, the images are stored in `imageList`, you can parse it and return the images to the user.
```suggestion
    const content = $('#articleContent');
    if (content.length) {
        return content.first().html()?.trim();
    }
    const video = $('#videoPlayer div');
    if (video.length) {
        return `<video controls preload="auto" poster="${video.attr('data-thumbnail')}"><source src="${video.attr('data-url')}" /></video>`;
    }
    const imageList = $('script:contains("imageList")').text();
    if (imageList) {
        const list = JSON.parse(imageList.match(/(\[.*\])/s)?.[1].replaceAll(/("description": ".*"),/g, '$1') || '[]');
        return list.map((item: any) => `<img src="${item.url}" alt="${item.description || ''}" />`).join('');
    }
    return '';
```